package.json

```diff
@@ -10,14 +10,14 @@
     "url": "https://github.com/nodejs/corepack.git"
   },
   "license": "MIT",
-  "packageManager": "yarn@4.0.0-rc.6",
+  "packageManager": "yarn@4.0.0-rc.13",
```

Comment:
```suggestion
  "packageManager": "yarn@4.0.0-rc.14",
```
Release is running now and will be ready in about 10 minutes

Reply:
Do you have its SHA by any chance?

Reply:
Not yet, https://github.com/yarnpkg/berry/actions/runs/2712633958